docs/_data/oauth-authentication.json

```diff
@@ -10,6 +10,16 @@
             "type": "string",
             "description": "A technical client's password"
         },
+        {
+            "name": "certificate",
+            "type": "string",
+            "description": "x509 certificate chain generated by xsuaa service"
+        },
+        {
+            "name": "privateKey",
+            "type": "string",
+            "description": "Private key related to the generated certificate"
```

Comment:
Private key of the generated certificate

Reply:
I suggest to let it be like this because in the documentation for mTLS it is written like here.